__tests__/app.test.js

```diff
@@ -148,6 +149,28 @@ describe("GET", () => {
             })
         })
     })
+    describe('GET /api/users', () => {
+        test('status 200 should return all users object with appropriate properties', () => {
+            return request(app)
+            .get('/api/users')
+            .expect(200)
+            .then(({body}) => {
+                body.users.forEach((user) => {
+                    expect(user).toHaveProperty('username', expect.any(String));
+                    expect(user).toHaveProperty('name', expect.any(String));
+                    expect(user).toHaveProperty('avatar_url', expect.any(String));
+                });
+            });
+        });
+        test('status code 404 with a message Not Found e.g /api/userito', () => {
+            return request(app)
```

Comment:
We will likely already have a test for all bad paths, if so this test probably auto passed which is an indication that it wasn't needed. If we don't already have that, we can keep this one and put it in its own describe block as it's not related to the functionality of this endpoint 👍